catalogs/catalog-lakehouse-iceberg/build.gradle.kts

```diff
@@ -136,7 +149,27 @@ tasks {
     dependsOn(copyCatalogLibs, copyCatalogConfig)
   }
 }
+// run  ./gradlew :catalogs:catalog-lakehouse-iceberg:calculateDependenciesSize at the root of the project
```

Comment:
the length of the cmd name `calculateDependenciesSize` is too long, and i wonder is it necessary?